doc/api.rst

```diff
@@ -49,19 +49,31 @@ Metrics
    :toctree: generated/
    :template: function.rst
    
-   metrics.regression_coverage_score
+   metrics.add_jitter
    metrics.classification_coverage_score
-   metrics.regression_mean_width_score
+   metrics.classification_coverage_score_v2
    metrics.classification_mean_width_score
+   metrics.classification_ssc
+   metrics.classification_ssc_score
+   metrics.cumulative_differences
    metrics.expected_calibration_error
-   metrics.top_label_ece
+   metrics.hsic
+   metrics.kolmogorov_smirnov_cdf
+   metrics.kolmogorov_smirnov_p_value
+   metrics.kolmogorov_smirnov_statistic
+   metrics.kuiper_cdf
+   metrics.kuiper_p_value
+   metrics.kuiper_statistic
+   metrics.length_scale
+   metrics.regression_coverage_score
    metrics.regression_coverage_score_v2
-   metrics.classification_coverage_score_v2
+   metrics.regression_mean_width_score
    metrics.regression_ssc
    metrics.regression_ssc_score
-   metrics.classification_ssc
-   metrics.classification_ssc_score
-   metrics.hsic
+   metrics.sort_xy_by_y
+   metrics.spiegelhalter_p_value
+   metrics.spiegelhalter_statistic
+   metrics.top_label_ece
```

Comment:
On the user side, I don't think users need access to the following functions: add_jitter, cumulative_differences, length_scale, sort_xy_by_y. I see them more as utilities for other metrics.

Do you think it's necessary to present them in the API?

Reply:
Agreed for `add_jitter` and `sort_xy_by_y` but I would leave `cumulative_differences` and `length_scale` as they are used in the examples.